specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2019-02-01-preview/EventGrid.json

```diff
@@ -3217,6 +3620,10 @@
                 "schemaUrl": {
                     "description": "Url of the schema for this event type.",
                     "type": "string"
+                },
+                "isInDefaultSet": {
+                    "description": "IsInDefaultSet flag of the event type.",
```

Comment:
Nit: Would be good to have a better description for this one.

Reply:
Updated.